processor/src/operations/io_ops.rs

```diff
@@ -523,21 +523,21 @@ mod tests {
 
     #[test]
     fn op_read() {
-        // reading from tape should push the value onto the stack
-        let mut process = Process::new_dummy_with_advice_tape(&[3]);
+        // reading from stack should push the value onto the stack
```

Comment:
nit: I'd change this to: "popping from the advice stack should push the value onto the operand stack"